deploy-manage/autoscaling/autoscaling-in-eck.md

```diff
@@ -18,7 +18,7 @@ Elasticsearch autoscaling requires a valid Enterprise license or Enterprise tria
 ::::
 
 
-ECK can leverage the [autoscaling API](https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-autoscaling) introduced in Elasticsearch 7.11 to adjust automatically the number of Pods and the allocated resources in a tier. Currently, autoscaling is supported for Elasticsearch [data tiers](/manage-data/lifecycle/data-tiers.md) and machine learning nodes.
+ECK can leverage the [autoscaling API](https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-autoscaling) introduced in Elasticsearch 7.11 to adjust automatically the number of Pods and the allocated resources in a tier. Currently, autoscaling is supported for Elasticsearch [data tiers](/manage-data/lifecycle/data-tiers.md) and machine learning nodes. ECK scales Elasticsearch data and machine learning tiers exclusively by scaling storage. CPU and Memory are scaled *relative* to the storage resource min/max settings, and not independently.
```

Comment:
> ECK scales Elasticsearch data and machine learning tiers exclusively by scaling storage

I don't think this is true, at least for the ML and frozen tiers for which ES returns memory requirements: https://www.elastic.co/guide/en/elasticsearch/reference/current/autoscaling-deciders.html

Reply:
Thank you @barkbay. I have read through the docs, and I have updated appropriately.

Reply:
@barkbay I had noted the Frozen tier at the end of this paragraph, but I've updated this again to try to clarify. If you have a suggestion for making this more clear (maybe a table would help?), I'm up for suggestions.